### PR TITLE
Adds a new arcade prize

### PR DESCRIPTION
**Pull request by Supermichael777**
:cl:
add: Added fake pits. Get them from the arcade machine.
/:cl:

Honk.
https://www.youtube.com/watch?v=xFntFdEGgws

### CODE REVIEW

File: code/game/turfs/simulated/floor/fancy_floor.dm [hunk 1]
```diff
@@ -123,6 +125,12 @@
 	burnt = 1
 	update_icon()
 
+/turf/open/floor/fakepit	
+	name = "chasm"
```
Comment (ChangelingRain) [inline code/game/turfs/simulated/floor/fancy_floor.dm]: remove this so you can tell it's fake by the name, same as the space tile

**Comment by GunHog** [comment]
I want to put these **ALL OVER THE MINING BASE.**

File: code/game/turfs/simulated/floor/fancy_floor.dm (hunk 1 above)
Comment (Supermichael777) [inline code/game/turfs/simulated/floor/fancy_floor.dm]: there is already a tell. plus you can just toss a pen or something on it to check.

Comment (Supermichael777) [inline code/game/turfs/simulated/floor/fancy_floor.dm]: but FIIIIIINNEEEEE

Comment (Iamgoofball) [inline code/game/turfs/simulated/floor/fancy_floor.dm]: yeah the description rats it out, it's fine as-is

Comment (XDTM) [inline code/game/turfs/simulated/floor/fancy_floor.dm]: Yeah the toy singulo only reveals itself with the description

**Comment by RandomMarine** [comment]
If you place these around a real chasm will they smooth with each other?

**Comment by Kromgar** [comment]
You sick fuck

**Comment by Supermichael777** [comment]
@RandomMarine probably not. But I could make it

**Comment by RandomMarine** [comment]
DEVILISH

File: code/game/machinery/computer/arcade.dm [hunk 2]
```diff
@@ -37,6 +37,7 @@
 		/obj/item/toy/talking/skeleton							= 2,
 		/obj/item/weapon/coin/antagtoken						= 2,
 		/obj/item/stack/tile/fakespace/loaded					= 2,
+		/obj/item/stack/tile/fakepit/loaded					= 2,
```
Comment (KorPhaeron) [inline code/game/machinery/computer/arcade.dm]: fix your spacing its triggering me

**Review by imtakingabreakdontatme — CHANGES_REQUESTED** [review]

**Comment by kilkun** [comment]
emagging it should turn it into a real chasm

**Comment by optimumtact** [comment]
Taking too long to respond to maintainers

**Comment by PKPenguin321** [comment]
Noooo

**Comment by optimumtact** [comment]
You can pick up this pr yourself by cloning his repo and the branch and then pushing to your own repo and making the required changes.

**Comment by PKPenguin321** [comment]
How to clone repo

**Comment by optimumtact** [comment]
git add remote supermichael https://github.com/Supermichael777/tgstation

git fetch supermichael

now you can work with it like the origin or upstream repos for tg

**Comment by PKPenguin321** [comment]
Neat, I'll have a look later

**Pull request closed without merging**